### PR TITLE
Add simulated wind support for EKF_TYPE_SITL and flightaxis

### PR DESCRIPTION
**Pull request by kd0aij**
Would be useful for simulating with ground truth wind speed.

### CODE REVIEW

**Comment by kd0aij** [comment]
LOITER mode: RF8 wind direction 270, speed 20kph = 5.6 m/sec
3 40 second intervals EKF_TYPE:10, 3, 2
wind direction and speed are consistent, 
image

File: libraries/SITL/SITL.h [hunk 1]
```diff
@@ -47,6 +47,7 @@ struct sitl_fdm {
     double range;           // rangefinder value
     Vector3f bodyMagField;  // Truth XYZ magnetic field vector in body-frame. Includes motor interference. Units are milli-Gauss.
     Vector3f angAccel; // Angular acceleration in degrees/s/s about the XYZ body axes
+    Vector3f wind;
```
Comment (OXINARF) [inline libraries/SITL/SITL.h]: Can you add a comment with the unit?

Comment (kd0aij) [reply libraries/SITL/SITL.h]: Done, thanks for the review.

**Comment by peterbarker** [comment]
These patches got applied via PRs merged elswhere.

Thanks!

**Comment by kd0aij** [comment]
@peterbarker Thanks for letting me know.

**Pull request closed without merging**